net-im/psi/psi-1.2.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PLOCALES="be bg ca cs de en eo es et fa fi fr he hu it ja kk mk nl pl pt pt_BR ru sk sl sr@latin sv sw uk ur_PK vi zh_CN zh_TW"
+PLOCALE_BACKUP="en"
+
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.xz
+	https://github.com/psi-im/psi-l10n/archive/1.2.tar.gz -> psi-l10n-${PV}.tar.gz"
+
+inherit eutils l10n multilib qmake-utils
```

Comment:
I don't see `multilib.eclass` in use here - and after converting to `eapply` you should also be able to drop `eutils.eclass` - please check.